packages/extension-api/src/extension-api.d.ts

```diff
@@ -1987,6 +1987,7 @@ declare module '@podman-desktop/api' {
       callback: (event: PullEvent) => void,
     ): Promise<void>;
     export function deleteImage(engineId: string, id: string): Promise<void>;
+    export function getImageInspect(engineId: string, id: string): Promise<ImageInspectInfo>;
```

Comment:
ImageInspectInfo is not defined in this .d.ts

Reply:
(and yes typecheck directive is not raising errors...)

Reply:
right, fixing 👍